citation-matching/citation-matching-core-code/src/main/java/pl/edu/icm/coansys/citations/data/CitationMatchingJob.java

```diff
@@ -0,0 +1,137 @@
+package pl.edu.icm.coansys.citations.data;
+
+import java.io.IOException;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.hadoop.io.BytesWritable;
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.io.Writable;
+import org.apache.hadoop.mapreduce.lib.output.SequenceFileOutputFormat;
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+import pl.edu.icm.coansys.citations.hashers.HashGenerator;
+import scala.Tuple2;
+
+/**
+ * 
+ * Citation matching job.
+ * 
+ * @author Łukasz Dumiszewski
+ */
+
+public class CitationMatchingJob {
```

Comment:
Yes, it's not the best name, but it was such before. I wouldn't like to change it now.